From dc6216298561bd6583306f35228a47215aa0f561 Mon Sep 17 00:00:00 2001
From: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
Date: Mon, 6 Apr 2020 14:27:01 +0300
Subject: [PATCH] scripts: upgrade scripts to use Python3

python command no longer exists in yocto dunfell. Use python3
instead.

Signed-off-by: Codrin Ciubotariu <codrin.ciubotariu@microchip.com>
---
 scripts/addpmecchead.py    | 2 +-
 scripts/bitfieldsparser.py | 2 +-
 scripts/pmecc_head.py      | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/addpmecchead.py b/scripts/addpmecchead.py
index 6e68e02..ba424e8 100755
--- a/scripts/addpmecchead.py
+++ b/scripts/addpmecchead.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import struct, sys
 
diff --git a/scripts/bitfieldsparser.py b/scripts/bitfieldsparser.py
index 01c80d2..9c89dec 100644
--- a/scripts/bitfieldsparser.py
+++ b/scripts/bitfieldsparser.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 from __future__ import print_function
 import struct, sys, json
diff --git a/scripts/pmecc_head.py b/scripts/pmecc_head.py
index da893e9..ec399c7 100644
--- a/scripts/pmecc_head.py
+++ b/scripts/pmecc_head.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import struct, sys, json, os
 from pprint import pprint
-- 
2.17.1

